tests/virt_autotest/host_upgrade_step3_run.pm

```diff
@@ -34,7 +34,7 @@ sub run {
     my $installed_product = get_var('VERSION_TO_INSTALL', get_var('VERSION', ''));
     my $upgrade_product   = get_required_var('UPGRADE_PRODUCT');
     my ($upgrade_release) = lc($upgrade_product) =~ /sles-([0-9]+-sp[0-9]+)/;
-    if (check_var('ARCH', 'x86_64') && $installed_product =~ /12-sp5/img && $upgrade_product =~ /15-sp2/img) {
+    if (get_var('ARCH', '') =~ /(x86_64|aarch64)/img && $installed_product =~ /12-sp5/img && $upgrade_product =~ /(15-sp2|15-sp3)/img) {
```

Comment:
'Utils::Architectures' and version_utils 'is_sle' are simple and easy to use here, such as
is_x86_64, is_aarch64, is_sle('=12-sp5').

And '(15-sp2|15-sp3)', you have to add for each product, why not use a wider scope? eg. 15sp2+

Reply:
Done